File: src/main/java/org/pixeltime/enchantmentsenhance/interfaces/SpawnFirework.java
package org.pixeltime.enchantmentsenhance.interfaces;

import org.bukkit.entity.Player;

public interface SpawnFirework {
     void launch(Player player, int fireworkCount);

     void launch(
            Player player,
            int fireworkCount,
            int fireWorkRounds,
            int delay);
}

File: modules/EnchantmentsEnhance-Plugin/src/main/kotlin/org/pixeltime/enchantmentsenhance/interfaces/SpawnFirework.kt
package org.pixeltime.enchantmentsenhance.interfaces

import org.bukkit.entity.Player

interface SpawnFirework {
    fun launch(player: Player, fireworkCount: Int)

    fun launch(
        player: Player,
        fireworkCount: Int,
        fireWorkRounds: Int,
        delay: Int
    )
}
